### PR TITLE
Webhook refactor

### PR DESCRIPTION
**Pull request by hyper-neutrino**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -8,82 +8,120 @@
...
 with open("../configurations/vybot.txt", "r") as f:
-  secret = f.read().strip()
+    secret = f.read().strip()
 
 hooks = {}
 
+
 def handler(room):
-  def _inner(activity):
-    if room != rid: return
```
Comment (hyper-neutrino) [inline bot.py:17]: test

Comment (hyper-neutrino) [inline bot.py:17]: test

**Review by hyper-neutrino — COMMENTED** [review]
a

**Comment by AMiller42** [comment]
ayy it do be lookin epic good now

**Review by AMiller42 — COMMENTED** [review]
now it be seeing my epic commentating

**Pull request closed without merging**